packages/core/src/components/menu/menuItem.tsx

```diff
@@ -126,21 +130,25 @@ export class MenuItem extends AbstractComponent<IMenuItemProps, IMenuItemState>
         if (hasSubmenu) {
             const measureSubmenu = (this.props.useSmartPositioning) ? this.measureSubmenu : null;
             const submenuElement = <Menu ref={measureSubmenu}>{this.renderChildren()}</Menu>;
-            const popoverClasses = classNames({
-                [Classes.ALIGN_LEFT]: this.state.alignLeft,
-            });
+            const popoverClasses = classNames(
+                Classes.MINIMAL,
+                Classes.MENU_SUBMENU,
+                popoverProps.popoverClassName,
+                { [Classes.ALIGN_LEFT]: this.state.alignLeft },
+            );
 
             content = (
                 <Popover
-                    content={submenuElement}
                     isDisabled={disabled}
                     enforceFocus={false}
                     hoverCloseDelay={0}
                     inline={true}
                     interactionKind={PopoverInteractionKind.HOVER}
                     position={this.state.alignLeft ? Position.LEFT_TOP : Position.RIGHT_TOP}
-                    popoverClassName={classNames(Classes.MINIMAL, Classes.MENU_SUBMENU, popoverClasses)}
                     useSmartArrowPositioning={false}
+                    {...popoverProps}
+                    content={submenuElement}
+                    popoverClassName={popoverClasses}
```

Comment:
`content` can't be overwritten. `popoverProps.popoverClassName ` is added to `popoverClassName `

Reply:
💯